src/openzaak/utils/__init__.py

```diff
@@ -15,12 +17,43 @@ def parse_isodatetime(val) -> datetime:
     return dateutil.parser.parse(val)
 
 
-def build_absolute_url(path: str, request: Optional[HttpRequest] = None) -> str:
+def get_openzaak_domain() -> str:
+    """
+    Obtain the domain/netloc of Open Zaak according to settings or configuration.
+    """
     from django.contrib.sites.models import Site
 
+    if settings.OPENZAAK_DOMAIN:
+        return settings.OPENZAAK_DOMAIN
+
+    warnings.warn(
+        "Deriving the domain from the Site configuration will soon be deprecated, "
+        "please migrate to the OPENZAAK_DOMAIN setting.",
+        PendingDeprecationWarning,
+    )
+    return Site.objects.get_current().domain
+
+
+def build_absolute_url(path: str, request: Optional[HttpRequest] = None) -> str:
     if request is not None:
         return request.build_absolute_uri(path)
 
-    domain = Site.objects.get_current().domain
-    protocol = "https" if settings.IS_HTTPS else "http"
-    return f"{protocol}://{domain}{path}"
+    domain = get_openzaak_domain()
+    _furl = furl(
+        scheme="https" if settings.IS_HTTPS else "http", netloc=domain, path=path,
+    )
+    return _furl.url
+
+
+def build_fake_request(*, method="get", **kwargs) -> HttpRequest:
+    from rest_framework.test import APIRequestFactory
```

Comment:
not doing this as a local import breaks a number of things because of drf.settings.api_settings usage :thinking: